cmd/mailchain/internal/http/handlers/send_message.go

```diff
@@ -253,7 +253,7 @@ func isValid(p *PostRequestBody, protocol, network string) error {
 	}
 
 	// TODO: be more general when getting key from hex
-	p.publicKey, err = secp256k1.PublicKeyFromHex(p.Message.PublicKey)
+	p.publicKey, err = secp256k1.PublicKeyFromBytes(p.Message.PublicKey)
```

Comment:
The new method is expecting a byte array not string, you will need to decode the string first using `hexutil.Decode` the result can be passed to `PublicKeyFromBytes`

Reply:
Thanks, I will fixed this.